internal/serverinstall/nomad.go

```diff
@@ -890,6 +990,24 @@ func (i *NomadInstaller) InstallFlags(set *flag.Set) {
 		Target: &i.config.consulServiceBackendTags,
 		Usage:  "Tags for the Waypoint backend service generated in Consul.",
 	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "nomad-volume-type",
+		Target: &i.config.volumeType,
+		Usage:  "Nomad volume type for the Waypoint server.",
```

Comment:
We should include possible valid values here if we can. Is it just `csi` and `host`?